### PR TITLE
Return the created devices as json.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/cli/cmds/fleet.toit [hunk 1]
```diff
@@ -448,17 +448,23 @@ add-devices parsed/cli.Parsed config/Config cache/Cache ui/Ui:
...
     with-devices-fleet parsed config cache ui: | fleet/FleetWithDevices |
       count.repeat:
-        fleet.create-identity
+        id := random-uuid
+        path := fleet.create-identity
+            --id=id
             --group=group
             --output-directory=output-directory
-        written-count++
+        written-ids["$id"] = path
```
Comment (kasperl) [inline src/cli/cmds/fleet.toit:460]: So you get a map from id to path out. Cute.

File: src/cli/cmds/fleet.toit [hunk 2]
```diff
@@ -448,17 +448,23 @@ add-devices parsed/cli.Parsed config/Config cache/Cache ui/Ui:
...
   finally:
-    if count == written-count or written-count > 0:
-      ui.info "Created $written-count identity file(s)."
+      ui.do --kind=Ui.RESULT: | printer/Printer |
+        printer.emit-structured
+          --json=: written-ids
+          --stdout=: | printer/Printer |
+              if count == written-ids.size or written-ids.size > 0:
```
Comment (kasperl) [inline src/cli/cmds/fleet.toit:466]: If find it hard to reason about the condition here. Maybe add a comment to explain what is going on?

If we've written the last one (count == written-ids.size) then it makes sense to report the number of files created. Why does it make sense to always repoted this if written-ids.size > 0?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/cli/cmds/fleet.toit (hunk 2 above)
Comment (floitsch) [reply src/cli/cmds/fleet.toit]: improved the code and added a comment.
done.

**Pull request merged**